From a98ac42321e53412aca9f84130305a53812bb4aa Mon Sep 17 00:00:00 2001
From: Bharat Bhushan <Bharat.Bhushan@nxp.com>
Date: Fri, 4 May 2018 14:36:29 +0800
Subject: [PATCH 326/448] vfio fsl-mc: Fix header file reference

Change header file reference as mc-bus driver moves
out of staging.

These changes will be squashed with other changes when
sending upstream.

Signed-off-by: Bharat Bhushan <Bharat.Bhushan@nxp.com>
---
 drivers/vfio/fsl-mc/vfio_fsl_mc.c      |    5 +----
 drivers/vfio/fsl-mc/vfio_fsl_mc_intr.c |    2 +-
 2 files changed, 2 insertions(+), 5 deletions(-)

diff --git a/drivers/vfio/fsl-mc/vfio_fsl_mc.c b/drivers/vfio/fsl-mc/vfio_fsl_mc.c
index 1ec2b31..9a356fc 100644
--- a/drivers/vfio/fsl-mc/vfio_fsl_mc.c
+++ b/drivers/vfio/fsl-mc/vfio_fsl_mc.c
@@ -19,10 +19,7 @@
 #include <linux/vfio.h>
 #include <linux/delay.h>
 
-#include "../../staging/fsl-mc/include/mc.h"
-#include "../../staging/fsl-mc/include/mc-bus.h"
-#include "../../staging/fsl-mc/include/mc-sys.h"
-#include "../../staging/fsl-mc/bus/dprc-cmd.h"
+#include <linux/fsl/mc.h>
 
 #include "vfio_fsl_mc_private.h"
 
diff --git a/drivers/vfio/fsl-mc/vfio_fsl_mc_intr.c b/drivers/vfio/fsl-mc/vfio_fsl_mc_intr.c
index eb244bb..b3ce07d 100644
--- a/drivers/vfio/fsl-mc/vfio_fsl_mc_intr.c
+++ b/drivers/vfio/fsl-mc/vfio_fsl_mc_intr.c
@@ -15,7 +15,7 @@
 #include <linux/eventfd.h>
 #include <linux/msi.h>
 
-#include "../../staging/fsl-mc/include/mc.h"
+#include "linux/fsl/mc.h"
 #include "vfio_fsl_mc_private.h"
 
 static irqreturn_t vfio_fsl_mc_irq_handler(int irq_num, void *arg)
-- 
1.7.1

